scripts/karma.coffee

```diff
@@ -10,7 +10,7 @@
 
 module.exports = (robot) ->
 
-  robot.hear /^@?(.*?)(\+\+|--).*?$/, (response) ->
+  robot.hear /@(.*?)(\+\+|--).*?$/, (response) ->
     thisUser = response.message.user
     targetToken = response.match[1].replace(/.*@/, '').trim()
```

Comment:
I'd check the match results and if there's more than one space just ignore and return.

Valid: `@Neal Stanard`
Invalid: `@Neal Stanard this is more than one space --`